### PR TITLE
Allow a custom login logo to be displayed on login

### PR DESCRIPTION
**Pull request by lukebarnard1**
This is to allow team logos to be shown when a user registers with a team email.

### CODE REVIEW

File: src/components/views/login/VectorLoginHeader.js [hunk 1]
```diff
@@ -23,11 +23,14 @@ module.exports = React.createClass({
...
     },
+    propTypes: {
+        icon: React.PropTypes.string,
+    },
 
     render: function() {
         return (
             <div className="mx_Login_logo">
-                <img src="img/logo.png" width="195" height="195" alt="Riot"/>
+                <img src={this.props.icon || "img/logo.png"} width="195" height="195" alt="Riot"/>
```
Comment (dbkr) [inline src/components/views/login/VectorLoginHeader.js]: Will icons always be square? May want to only set one of these dimensions and give it a bounding box if necessary to stop things around it jumping about after the image loads.

Comment (lukebarnard1) [inline src/components/views/login/VectorLoginHeader.js]: Oh, two good points.

Comment (dbkr) [inline src/components/views/login/VectorLoginHeader.js]: Come to think of it, I'd have thought you'd still want the riot logo in any case, and the team logo would be in addition?

Comment (lukebarnard1) [inline src/components/views/login/VectorLoginHeader.js]: The current design is to replace it entirely

**Pull request merged**